gempak/ush/gfs_meta_sa2.sh

```diff
@@ -34,30 +28,25 @@ device="nc | ${metaname}"
 # IF CYCLE IS NOT 00Z OR 06Z EXIT SCRIPT.
 # Also exit if run from 00z gfs
 #
-if [ ${cyc} -eq "12" ] || [ ${cyc} -eq "18" ] 
-then
-    exit
-# elif [ ${cyc} -eq "00" ] && [ $(echo $COMIN | awk -F/ '{print $5}' | awk -F. '{print $1}') = "gfs" ]
-elif [ ${cyc} -eq "00" ] && [ ${mdl} = "gfs" ]
-then 
-    # don't want to run from 00z gfs
+if [[ ${cyc} != "06" ]]; then
     exit
 fi
 
-PDY2=$(echo ${PDY} | cut -c3-)
-# export HPCGFS=$COMROOT/nawips/${envir}/gfs.${PDY}
-export HPCGFS=${COMINgempak}/${mdl}.${PDY}/${cyc}/${COMPONENT}/gempak
-
-grid1="F-GFSHPC | ${PDY2}/${cyc}00"
+grid1="F-GFSHPC | ${PDY:2}/${cyc}00"
 
 # DEFINE YESTERDAY
-PDYm1=$($NDATE -24 ${PDY}${cyc} | cut -c -8)
-PDY2m1=$(echo ${PDYm1} | cut -c 3-)
+PDYm1="$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - 24 hours")"
 
-$GEMEXE/gdplot2_nc << EOF
+for symlink in ecwmf_m1 ukmet; do
```

Comment:
This should be:
for symlink in ec**mw**f_m1 ukmet; do

Reply:
Fixed.